xcodeproj/internal/default_automatic_target_processing_aspect.bzl

```diff
@@ -90,6 +90,7 @@ def _default_automatic_target_processing_aspect_impl(target, ctx):
     elif ctx.rule.kind == "objc_library":
         xcode_targets = {
             "deps": [target_type.compile, None],
+            "implementation_deps": [target_type.compile],
```

Comment:
I feel like this maybe warrants a note as it's not available in Bazel and someone might just remove it in the future thinking it's not used? Also wondering if there are plans to open-source the patch or upstream it, it sounds like it might be useful!

Reply:
We can definitely put up the PR with documentation against Bazel upstream, but are not optimistic of it making it in since the Bazel team wants to migrate starlark implementation (our implementation is based on existing java code). But a comment would be nice to give context.

Reply:
@yongjincho92 After you add the comment, we can merge this 🙇.

Reply:
sorry for the delay. Was waiting for the draft pr on bazel upstream to be uploaded. Is it okay I add comment without a link for now?

Reply:
Yeah, no need for the PR link, just a comment explaining what it's doing is fine.